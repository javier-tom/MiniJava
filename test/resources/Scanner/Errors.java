int five = 5 /* 4;

_eleven