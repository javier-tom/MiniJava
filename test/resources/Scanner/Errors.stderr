
Unexpected character '/' on line 1 at column 14 of input.

Unexpected character '_' on line 3 at column 1 of input.